tests/unit/test_configuration.py

```diff
@@ -469,3 +469,17 @@ def test_no_noise():
             column_noise_dict = dataset_column_dict[column]
             for column_noise_type in column_noise_dict.keys():
                 assert column_noise_dict[column_noise_type][Keys.CELL_PROBABILITY] == 0.0
+
+
+@pytest.mark.parametrize(
+    "dataset_name",
+    [x.name for x in DATASETS] + [None],
+)
+def test_get_config_dataset_name_key(dataset_name):
+    """Tests that the dataset name is returned as the first key"""
+    outer_keys = set(get_config(dataset_name).keys())
+    if dataset_name:
+        assert outer_keys == {dataset_name}
+    else:
+        # TODO: Convert pseudopeople.constants.metadata::DatasetNames to a NamedTuple
+        assert outer_keys == {x.name for x in DATASETS}
```

Comment:
Do we have a test that an error is thrown if a non-existent dataset name is provided?

Reply:
now we do